### ISSUE TITLE
Exception in send causing thread to stop...

### ISSUE THREAD

**Issue by daid** [jstasiak/python-zeroconf#19]
Just quickly dumping these here while I have the backtrace.

Exception in thread Thread-11:
Traceback (most recent call last):
  File "C:\Software\Cura\python\lib\threading.py", line 552, in __bootstrap_inner
    self.run()
  File "C:\Software\Cura_WiFi_Dev\zeroconf\zeroconf.py", line 1061, in run
    self.zc.send(out)
  File "C:\Software\Cura_WiFi_Dev\zeroconf\zeroconf.py", line 1661, in send
    bytes_sent = s.sendto(packet, 0, (addr, port))
error: [Errno 10065] Er is geprobeerd een socketbewerking uit te voeren op een onbereikbare host

Not sure what the implications are. But I got this exception somewhere today while running the python-zeroconf. (On windows, on bad-wifi)

**Comment by jstasiak** [comment]
Hey

I think the consequence will be the service discovery stopping to work which is undesired (the workaround I'd suggest is to manually make sure ``ServiceBrowser`` instance you use hasn't crashed using its ``is_alive`` method (it inherits Thread at the moment)).

I think I have an idea how to reproduce something similar, I'll let you know if I find anything useful.

**Comment by jstasiak** [comment]
I failed to reproduce this so far, is it still happening to you @daid?

**Comment by daid** [comment]
Very rarely. It seems to be related to having python-zeroconf running for a long time, and connecting/disconnecting to different wifi networks. (especially open wifi spots that require you to sign in)

**Comment by gustavotemple** [comment]
I'm having the same issue:
```
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 1060, in run
    self.zc.send(out)
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 1670, in send
    bytes_sent = s.sendto(packet, 0, (addr, port))
error: [Errno 101] Network is unreachable
```

My Linux:
```
$ uname -a
Linux 3.13.0-88-generic #135-Ubuntu SMP Wed Jun 8 21:10:42 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by stephenrauch** [comment]
@gustavotemple 

Can you open a new issue and give more info on how this occurred? This bug was not reproducible.

Thanks.

**Comment by stephenrauch** [comment]
Closed via #72
